src/RecaptchaServiceInterface.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\ReCaptcha;
+
+/**
+ * An interface for interacting with a recaptcha service provider
+ */
+interface RecaptchaServiceInterface
```

Comment:
Thank you for the review!

[`laminas-captcha`](https://github.com/laminas/laminas-captcha/blob/2.18.x/src/ReCaptcha.php) has calls to all of the functions in the interface, I think because it can create the service by itself if the options are provided.

I'll remove the types as a start :)